pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+## Pull request type
+Please check the type of change your PR introduces:
+- [ ] Bugfix
+- [ ] Feature
+- [ ] Code style update (formatting, renaming)
+- [ ] Refactoring (no functional changes, no api changes)
+- [ ] Build related changes
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Other (please describe):
+
+## Problem/feature
+
+_What problem are you trying to solve?_
```

Comment:
Do we want to add this line in here so that when we add the issue number, it will link the pull request comment to the GitHub Issue? `Fixes # (issue)`